### ISSUE TITLE
about lfw accuracy

### ISSUE THREAD

**Issue by k7k8k91** [justinshaohi/gluon_mobilefacenet#1]
hello，I run your code, find that the highest accuracy in lfw is about 98.9%.  But the insightface code can easily get 99.5% accuracy in lfw. Is there anying wrong in the training code? or the evaluation method is different? Can you provide your email? So I can contact you easily. Thank you!

**Comment by justinshaohi** [comment]
@k7k8k91 hi ,i just use L2softmax to train the model,and my devices only has one 1080ti gpu. so,if you want to improve accuary,use arcface loss and use big batch（like 4 gpu 512 batch size）,it can help u to improve a part of accuracy
